src/Ocelot.Provider.Polly/PollyPoliciesDelegatingHandler.cs

```diff
@@ -30,27 +33,21 @@ private IPollyQoSProvider<HttpResponseMessage> GetQoSProvider()
         return _contextAccessor.HttpContext.RequestServices.GetService<IPollyQoSProvider<HttpResponseMessage>>();
     }
 
-    protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request,
-        CancellationToken cancellationToken)
+    /// <summary>
+    /// Sends an HTTP request to the inner handler to send to the server as an asynchronous operation.
+    /// </summary>
+    /// <param name="request"></param>
+    /// <param name="cancellationToken"></param>
+    /// <exception cref="BrokenCircuitException"></exception>
+    /// <exception cref="HttpRequestException"></exception>
+    /// <returns></returns>
```

Comment:
You need to complete these empty XML-docs lines.
Otherwise just remove empty tags.